client/package.json

```diff
@@ -1,71 +1,47 @@
 {
 	"name": "veryrezsi",
-	"version": "0.0.2",
-	"type": "module",
+	"version": "0.0.1",
+	"private": true,
 	"scripts": {
-		"dev": "vite dev --port 4000",
+		"dev": "vite dev",
```

Comment:
Adding _--port 3000_ to the end would help, random ports are not fun to me.

Reply:
Hopefully Vite plugin can follow this.

Well Vite plugin does not give a 🍆, it uses _npx vite --port=4000_ to start dev server.

Reply:
I think it is only random if something is blocking the port, if not then it is 5173.  I will set it to 4000 because 3000 is blocked for me by a note taking app I use.